content/developer/reference/javascript/hooks.rst

```diff
@@ -47,3 +49,59 @@ API
     :param string eventName: the name of the event that we want to listen to
     :param function callback: listener callback
 
+.. _jsref/position_hook:
+
+usePosition
+===========
+
+Location
+--------
+
+`@web/core/position/position_hook`
+
+Description
+-----------
+
+Helps positioning a (piece of) component relatively to an HTMLElement through a ``position: fixed;`` design strategy. This hook ensures the positioning is updated when the window is resized/scrolled.
+
+.. code-block:: javascript
+
+    import { usePosition } from "@web/core/position/position_hook";
+
+    class MyPopover {
+      setup() {
+        // Here, the target is an HTMLElement
+        usePosition(this.props.target);
+      }
+    }
+    MyPopover.template = owl.tags.xml`<div>I am positioned through a wonderful hook!</div>`
+
+API
+---
+
+.. js:function:: usePosition(reference[, options])
+
+    :param HTMLElement,()=>HTMLElement reference: the reference HTMLElement to be positioned from
+    :param Options options: the positioning options
+
+Options
+~~~~~~~
+.. list-table::
+   :widths: 20 20 60
+   :header-rows: 1
+
+   * - Name
+     - Type
+     - Description
+   * - ``popper``
+     - string
+     - this is the element that will get positioned. You can provide here a `useRef reference <https://github.com/odoo/owl/blob/master/doc/reference/hooks.md#useref>`_. If not provided, ``this.el`` is used (default: not provided).
+   * - `container`
+     - HTMLElement
+     - the container from which the popper is expected not to overflow. If overflowing occurs, other popper positions are tried until a not overflowing one is found. (default: the `<html/>` node)
+   * - ``margin``
+     - number
+     - added margin between popper and reference elements (default: `0`)
+   * - ``position``
+     - string
+     - the desired position. It is a string composed of one direction and one variant separated by a dash character. Valid directions are: ``top``, ``bottom``, ``right``, ``left``. Valid variants are: ``start``, ``middle``, ``end``. The variant can be omitted (default variant is ``middle``). Examples of valid positions: ``right-end``, ``top-start``, ``left-middle``, ``left``. (default position: `bottom`)
```

Comment:
should we add a note about the current limitation of being positioned with a `fixed` strategy? I don't remember the details exactly, but I believe there is a limitation